extensions/reviewed/ExtendedVariables.json

```diff
@@ -1072,7 +981,7 @@
         {
           "codeOnly": false,
           "defaultValue": "yes",
-          "description": "Clear destination first",
+          "description": "Exact copy?",
           "longDescription": "",
           "name": "isExactCopy",
```

Comment:
Why this change? What does this parameter do?

Reply:
Because it's unclear.
Exact copy mean that copy everything, it does not append or prepend stuff to the existing variable.

Reply:
`Clear destination first` seems to better describe what you've just said.
I'm not against a rephrasing though.
Also, isn't `?` against the good practices?

Reply:
`Clear the variable before copying` ?

Reply:
Ok, I changed to 
`Clear the destination variable before copying`